app/internal/mail.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from fastapi.templating import Jinja2Templates
+from fastapi_mail import ConnectionConfig, FastMail, MessageSchema
+from pydantic import EmailStr
+from pydantic.errors import EmailError
+
+import config
+
+templates = Jinja2Templates(directory=os.path.join("app", "templates"))
+
+# application name
+CALENDAR_SITE_NAME = "Calendar"
+# link to the home page of the application
+CALENDAR_HOME_PAGE = "calendar.com"
+# link to the application registration page
+CALENDAR_REGISTRATION_PAGE = r"calendar\registration.com"
```

Comment:
The top-level domain (.com) should be placed right after the host (i.e. calendar.com/registration).
consider app.url_oath_for() here as well.

Reply:
ok, I will fix the top-level domain. thanks

regarding the rest see above